src/extractor/extraction_containers.cpp

```diff
@@ -979,6 +982,54 @@ void ExtractionContainers::PrepareTrafficSignals(
     log << "ok, after " << TIMER_SEC(prepare_traffic_signals) << "s";
 }
 
+// TODO: copy-paste
+void ExtractionContainers::PrepareStopSigns(const ReferencedStopSigns &referenced_stop_signs) {
```

Comment:
Can probably be combined with the traffic signal functions.

Reply:
Sure, I am going to refactor it a lot, just want to have preliminary "green light" about approach.

Reply:
Yep, looks good :green_circle: